modin/experimental/cloud/test/test_cloud.py

```diff
@@ -33,7 +33,8 @@ def test_update_conda_requirements(setup_commands_source):
     major = sys.version_info.major
     minor = sys.version_info.minor
     micro = sys.version_info.micro
-    python_version = f"python=={major}.{minor}.{micro}"
+    python_version_lower = f"python>{major}.{minor}"
+    python_version_higher = f"python<={major}.{minor}.{micro}"
```

Comment:
maybe we should mock `sys.version_info` to some fixed bogus value like `7.12.45` and check for that string?

Reply:
ready